src/component/Counter/Counter.test.js

```diff
@@ -0,0 +1,30 @@
+import { render, fireEvent } from "@testing-library/react";
+import Counter from "./Counter";
+
+describe("Counter Component", () => {
+  it("Should render with initial count", () => {
+    const { getByText } = render(<Counter initialValue={5} />);
```

Comment:
You can also think of encapsulating render logic, thus also reducing redundancy
`
setUp(props){ 
  return render(<Counter {...props}}>)
}
  //Inside tests 
const {getBytext} = setUp(props)....
`

Reply:
Fixed as part of 903b467